src/main/java/com/microsoft/sqlserver/jdbc/dns/DNSKerberosLocator.java

```diff
@@ -0,0 +1,37 @@
+package com.microsoft.sqlserver.jdbc.dns;
```

Comment:
can you add the licence header files to the new classes? You can find them in other classes.